org.ekstep.navigation-1.0/renderer/plugin.js

```diff
@@ -77,6 +78,9 @@
   initPlugin: function (data) {
       // Plugin actions are handled in the angularJS controller.
   },
+  resetNavigation: function(){
```

Comment:
Move this function to the controller. Both event listener & function should be in the controller.